components/bounty/bounty-hero/bounty-settings/controller.tsx

```diff
@@ -46,10 +53,14 @@ export default function BountySettings({
   };
 
   async function handleHardCancel() {
-    handleHardCancelBounty(currentBounty.contractId, currentBounty.issueId)
+    handleHardCancelBounty(currentBounty.contractId, currentBounty.id)
       .then(() => {
         updateWalletBalance();
         updateBountyData();
+      })
+      .catch(error => {
+        dispatch(toastError(t("errors.something-went-wrong"), t("actions.failed")));
```

Comment:
it was not "something", it was "bounty could not be created" -- if we are adding visual errors, they might as well be descriptive :+1: